### PR TITLE
EPS-211 unable to upload bigger file

### PR DESCRIPTION
**Pull request by ondrej-kvasnovsky**
increased limits to 10mb, defaults were:  jsonLimit 1mb, formLimit 56…kb and textLimit 56kb

more info: https://github.com/dlau/koa-body

### CODE REVIEW

File: lib/http_server/server.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ module.exports = class {
       const koaBody = require('koa-body')
       this.koa.use(
         koaBody({
+          jsonLimit: '10mb',
```
Comment (igorshapiro) [inline lib/http_server/server.js]: Can we make it configurable?

Say, using `config.get('quadro.http.koaBody.options')`?

Comment (tomeresk) [reply lib/http_server/server.js]: 👍

Comment (ondrej-kvasnovsky) [reply lib/http_server/server.js]: Ok, it is changed now.

File: lib/http_server/server.js [hunk 2]
```diff
@@ -19,11 +19,13 @@ module.exports = class {
 
     if (enableKoaBodyParser) {
       const koaBody = require('koa-body')
-      this.koa.use(
-        koaBody({
-          multipart: true
-        })
-      )
+      const options = await this.config.get('quadro.middleware.koa-body.options', {
```
Comment (tomeresk) [inline lib/http_server/server.js]: I would suggest merging the options from the config with the default options.
This way if someone wants to change or add a specific option they would not lose the rest of the defaults.

For example:

```js
const defaultOptions = {
  jsonLimit: '10mb',
  textLimit: '10mb',
  formLimit: '10mb',
  multipart: true
}
const configOptions = await this.config.get('quadro.middleware.koa-body.options', {})
const options = { ...defaultOptions, ...configOptions }
```

**Pull request merged**